force-app/tests/Trigger Recipes/AccountTriggerHandler_Tests.cls

```diff
@@ -12,7 +12,7 @@ private inherited sharing class AccountTriggerHandler_Tests {
         Test.stopTest();
 
         Account check = [SELECT Id, Description FROM Account LIMIT 1];
-        System.assertEquals(
+        System.Assert.AreEqual(
```

Comment:
I see that Kevin mixed the order of the arguments in a lot of these tests.
The correct argument order for assertions is `expected` first then, `actual` and `message`.
This means that for this example, it should be:
```js
Assert.areEqual(1, Integer.valueOf(check.Description), 'Expected...');
```

Reply:
On a separate note:
- can we get rid of all of the `System` namespace to make the code less verbose (unless you have a strong opinion against it)?
- please keep the method names in camel case starting with a lower case: `areEqual`

Reply:
@pozil Good catch on **camel casing** and the order in which asserts where written. I will fix it.

On using `System` namespace, I am with you on code less verbose but unfortunately we could have an org when someone already created a class named `Assert`, and then our code fails to deploy or work. This is a rare possibility but since Salesforce allows one to create a class named `Assert`, I think using the System namespace keeps us shielded from this rare scenario.

Reply:
The camel casing and the order of how assertions were written are fixed. Please review.